specification/web/resource-manager/Microsoft.Web/2016-03-01/ResourceProvider.json

```diff
@@ -307,6 +307,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/verifyHostingEnvironmentVnet": {
+      "post": {
+        "summary": "Verifies if this VNET is compatible with an App Service Environment by analyzing the Network Security Group rules",
```

Comment:
I shortened the summary in the next iteration.
Our tooling auto generates this swagger file hence why the summary and description are the same. Similar to the other APIs in the file.